genomic_data_service/region_indexer_encode.py

```diff
@@ -0,0 +1,140 @@
+from genomic_data_service.region_indexer_task import index_file
+from genomic_data_service.region_indexer_elastic_search import RegionIndexerElasticSearch
+
+import requests
+
+from os import environ
+
+
+if 'ES' in environ:
+    es_uri = [environ['ES']]
+else:
+    es_uri = ['localhost']
+es_port = 9201
+
+
+SUPPORTED_CHROMOSOMES = [
+    'chr1', 'chr2', 'chr3', 'chr4', 'chr5', 'chr6', 'chr7', 'chr8', 'chr9',
```

Comment:
Formatting nit. Usually like to put one item per line for long lists.

Reply:
yes, it would be way too long though, maybe refactor to generate the strings based on the number of chromosomes supported